templates/al2023/runtime/gpu/initialize-nvidia-clock.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+if ! gpu-ami-util has-nvidia-devices; then
+  echo >&2 "no NVIDIA devices are present, nothing to do!"
+  exit 0
+fi
+
+# gpu boost clock
+if command -v nvidia-smi &> /dev/null; then
+  echo >&2 "INFO: nvidia-smi found"
+
+  nvidia-smi -q > /tmp/nvidia-smi-check
```

Comment:
is this existence check needed anymore? if it doesn't exist then the unit fails and that doesn't hurt anything does it?

Reply:
yes, that's some leftovers, `nvidia-smi` should always be present on nvidia ami